faststream/cli/supervisors/multiprocess.py

```diff
@@ -38,3 +40,32 @@ def shutdown(self) -> None:
             process.join()
 
         logger.info(f"Stopping parent process [{self.pid}]")
+
+    def restart(self) -> None:
+        active_processes = []
+
+        for process in self.processes:
+            if process.is_alive():
+                active_processes.append(process)
+                continue
+
+            pid = process.pid
+            exitcode = process.exitcode
+
+            log_msg = "Worker (pid:%s) exited with code %s."
+            if exitcode and abs(exitcode) == signal.SIGKILL:
+                log_msg += " Perhaps out of memory?"
+            logger.error(log_msg, pid, exitcode)
+
+            process.kill()
+
+            new_process = self._start_process()
+            logger.info(f"Started child process [{new_process.pid}]")
+            active_processes.append(new_process)
+
+        self.processes = active_processes
+
+    def should_restart(self) -> bool:
```

Comment:
Probably we can use smth like `not all((p.is_alive() for p in process))`? It looks much explicit for me

Reply:
I agree, it looks better. Fixed